build.gradle

```diff
@@ -4,18 +4,23 @@ plugins {
     id 'com.github.jruby-gradle.base' version '0.1.5'
     id 'java'
 }
+
 import com.github.jrubygradle.JRubyExec
 
 allprojects {
     group = 'org.embulk.input.jdbc'
     version = '0.6.0'
+
+    apply plugin: 'jacoco'
+    jacoco {
+        toolVersion = '0.7.1.201405082137'
```

Comment:
Is this necessary? Can't we use the latest version every time by removing this section?